mergify_engine/branch_updater.py

```diff
@@ -168,7 +168,12 @@ def update_with_api(pull):
             input={"expected_head_sha": pull.g_pull.head.sha},
             headers={"Accept": "application/vnd.github.lydian-preview+json"})
     except github.GithubException as e:
-        if e.status < 500:
+        if e.status == 422:
+            LOG.debug("update branch done somewhere in meantime",
```

Comment:
I'd go for "branch already updated" or "already up-to-date"

Reply:
Well maybe not, it can also be some-else that push more commits in the meantime

Reply:
Ah ok I see what you mean. So maybe "branch updated in the meantime" would be clearer.

Reply:
done